### PR TITLE
a small start to the integrations section of the config

### PR DESCRIPTION
**Pull request by freben**
A small step toward #4310

### CODE REVIEW

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4561-6d3dsqrsvbygwmywmxgpdrvba0n6j0eg.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/602d0c75fcb22d056f57b16d

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr4561-6d3dsqrsvbygwmywmxgpdrvba0n6j0eg.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/602d0c75fcb22d056f57b16d

**Comment by Fox32** [comment]
Very cool. I guess I should add one for the Microsoft Graph import

**Comment by freben** [comment]
All contributions welcome! Let's grow this thing.

Also kinda thinking that maybe each integration could have an overview page as well, where it perhaps can show a clear overview of what things are supported and not supported.

Something like tech X having a checkbox for catalog, but cross mark (and link to relevant issue to track) for scaffolder

File: docs/integrations/github/org.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+id: org
+title: GitHub Organizational Data
+sidebar_label: Org Data
+# prettier-ignore
```
[the remark this answers is not in the document]
Comment (Fox32) [reply docs/integrations/github/org.md:5]: Prettier might break long descriptions into multiple lines like you would normally do it for yaml. But docusaurus doesn't support that and would only show a portion of the description 😞

[the remark this answers is not in the document]
Comment (Fox32) [reply docs/integrations/github/org.md:5]: More about that here: #3927

**Review by adamdmharvey — COMMENTED** [review]
Would have found this super useful when I first came across Backstage! Thanks for putting together. I think I have some notes on my internal repo I could try to share out here as well, will give a look.

File: docs/integrations/github/org.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (freben) [reply docs/integrations/github/org.md:5]: Yeah I'm making it a habit to paste that thing in there essentially for the future... So it doesn't break when some day somebody extends the description and it line breaks

File: docs/integrations/github/org.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+The example above assumes that the backend is started with an environment
+variable called `GITHUB_TOKEN` that contains a Personal Access Token. The token
+needs to have at least the scopes `read:org`, `read:user`, and `user:email` in
+the given `target`.
+
+If you want to address your own GitHub Enterprise instance, replace occurrences
+of `https://github.com` in the configuration above with the address of your
+GitHub Enterprise home page, and the `apiBaseUrl` to where your API endpoint
+lives - commonly on the form `https://<host>/api/v3`.
```
[the remark this answers is not in the document]
Comment (freben) [reply docs/integrations/github/org.md:71]: The whole config structure should be refactored - it's a bit awkwardly placed in the hierarchy, and was added long before the `integrations` craze so it doesn't use host matching etc. And regarding apiBaseUrl et al, it should definitely sideload that out of the integrations if present; no need to specify doubles everywhere

File: docs/integrations/ldap/org.md [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+  processors:
+    ldapOrg:
+      providers:
+        - target: ldaps://ds.example.net
+          bind:
+            dn: uid=ldap-reader-user,ou=people,ou=example,dc=example,dc=net
+            secret:
+              $env: LDAP_SECRET
+          users:
+            dn: ou=people,ou=example,dc=example,dc=net
```
[the remark this answers is not in the document]
Comment (freben) [reply docs/integrations/ldap/org.md:52]: Heh, I'm sure you are correct, this was a bit of a layman's implementation in the first place :) could probably do one of those responsible refactors where we still support the old deprecated way for a long time if the user used that format

File: docs/integrations/github/org.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (benjdlambert) [reply docs/integrations/github/org.md:5]: Well, how annoying.

**Review by benjdlambert — APPROVED** [review]

File: docs/integrations/github/org.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Fox32) [reply docs/integrations/github/org.md:5]: As far as I understand, it should be fixed in newer versions of docusaurus, where #3197 comes into play

**Pull request merged**